content/ja/handbook/engineering/development/growth/experimentation.md

```
@@ -0,0 +1,170 @@
+### 実験プロセス
+
+#### 概要
+
+1. 実験のスコープを定義するための[実験アイデア](https://gitlab.com/gitlab-org/gitlab/-/issues/new?issuable_template=Experiment%20Idea) Issue を作成します（追加ラベル: `~"experiment idea"`、任意で追加: `~"growth experiment"`）
+
+* 実験として試みるアイデアを持つ人は誰でもこの Issue を作成できます
+* 実験したいプロダクトの部分、実験のアイデア（つまり仮説）、成功した結果がどのようなものかについての情報を提供します
+* この非常に初期の段階で最も重要なことは、チームの他のメンバーが理解でき、プロジェクトマネージャーが引き継げるだけの十分なコンテキストと詳細でアイデアをメモしておくことです
+
+1. プロジェクトマネージャー（PM）は[実験バックログ](https://gitlab.com/groups/gitlab-org/-/boards/2028884)を定期的にレビューして、明確に定義された実験の必要な基準を満たしているかを確認します
+    * [実験定義標準](#experiment-definition-standards)を満たし、関連性があると判断された実験は、適切なマイルストーンリスト（`%Awaiting further demand`、`%Backlog`、`%Next 1-3 releases`、または特定のマイルストーン）を追加することで優先順位付けされます
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)